src/components/UI/atoms/Icon/Icon.jsx

```diff
@@ -89,6 +92,32 @@ display: ${display};
 margin: ${theme.spacing[margin]};
 margin-top: ${theme.spacing[marginTop]};
 margin-left: ${theme.spacing[marginLeft]};
+margin-right: ${theme.spacing[marginRight]};
+
+@media ${theme.device.mobileS} {
+  width: 1.3rem;
+  height: 1.3rem;
+  margin-left: ${marginLeft ? theme.spacing.xxxs : 0};
```

Comment:
Change to `margin-left: ${marginLeft && theme.spacing.xxxs};`

Reply:
&& returns `false` if the first parameter is undefined;
we will now have something like `margin-left: false` or `margin-left: undefined`